### ISSUE TITLE
Actual usage

### ISSUE THREAD

**Issue by RickCogley** [cssrecipes/defaults#6]
Hello - are there any real-world examples of the cssrecipes in use? 

I see that putaindecode is using some of them, but statinamic is a little hard to parse if you're not using it to implement. 

If there are any repos you might point me to, I'd be appreciative. 

Sincerely,.
Rick

**Comment by MoOx** [comment]
For this one (defaults), you just need to include it (import) in your pages, and that should work.
Feel free to ask for more if you need.

**Comment by RickCogley** [comment]
Hi, thanks, sorry to have asked on defaults. It was a general question to see if there are any repositories you know of, where someone is using a number of the cssrecipes in various ways. I am just trying to see how they work, in general. The tooltip one is working just fine now.

**Comment by MoOx** [comment]
README of each recipes should be enough.
If you have question(s) for a recipe, feel free to open an issue on the relevant repo :)
